test/cypress/integration/lazy-chat.test.js

```diff
@@ -0,0 +1,46 @@
+import { crispWebsiteId } from '../../../src/client/static/data/app.json'
+
+const lazyChatButtonIdentity = '[data-test="lazyChatButtonIdentity"]'
+const lazyChatOptInIdentity = '[data-test="lazyChatOptInIdentity"]'
+
+describe('Lazy chat', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('is only present if Crisp ID is set', () => {
```

Comment:
Use a variable so other devs directly see both the if and the else apply to the same element
```js
const button = cy.get(lazyChatButton)
if (chatIsEnabled) {
  button.should('be.visible')
} else {
  button.should('not.exist')
}
```